### ISSUE TITLE
can’t set correct keyboard layout (physical keyboard)

### ISSUE THREAD

**Issue by pharaonic-faery** [Divested-Mobile/DivestOS-Build#270]
- DivestOS 18.1
- November 13th 2023 update
- device: Redmi Note 7 (lavender)

Hello!
I’d like to use a physical keyboard instead of on-screen ones, but it's impossible to set the correct keyboard layout (I don’t use U.S. QWERTY keyboards).
(I had the same problem previously when LineageOS 18.1 was running on this device. I also have LineageOS 20 running on another device, and on that one, I can set the correct keyboard layout for my physical keyboards. So I guess the solution to this issue may exist somewhere in more recent versions of DivestOS or LineageOS)

Thanks! ☺

**Comment by SkewedZeppelin** [comment]
Works for me.

Settings > System > Language & input > Physical keyboard > [Name of the keyboard] > Set up keyboard layours > choose what you want

**Comment by pharaonic-faery** [comment]
Yes, this is what I do. But it doesn't work. The layout I select is not taken into account. It remains a qwerty layout.
